### DISCUSSION TITLE
Setting up Time Series data and WMTS

### DISCUSSION THREAD

**Discussion by sleagle** [TerriaJS/terriajs#6264] [Q&A]
Hi,

I am new to Terria.io/Maps and I am trying to set up a Time series data and also add WMTS layer.

- WMTS Configuration:
`{
    "name": "SGD2",
    "type": "wmts",
    "url": "/geoserver/gwc/service/wmts",
    "parameters": [
      {
        "layer": "vdl_map_to_ref_modified3",
        "style": "default",
        "service":"WMS",
        "tileMatrixSetID": "EPSG:900913",
        "crs":"EPSG:900913",
        "request":"GetLegendGraphic",
        "format":"image/png"
      }
    ],
    "opacity": 1
  }`
I am getting the following warning: Warning: This WMTS layer has no Web Mercator tile matrix set - it may not display correctly

- Time Series Configuration:

`{
    "type": "wfs",
    "name": "Published Locations",
    "url": "https://convictlandscapes.com.au/geoserver/convictlandscapes/ows",
    "parameters": [
      {
        "service": "WFS",
        "authkey":"ce85c5b3292a4e33a81a0222a1a7d7da",
        "sortBy":"occupy_start",
        "version":"2.0.0",
        "srsName":"EPSG:3857",
        "outputFormat":"application/json",
        "request":"GetCapabilities"
      }
    ],
    "description": "This is test information",
    "typeNames": "locations_published",
    "maxFeatures": 1000,
    "timeLabel": "Date Range",
    "initialTimeSource":"start",
    "startTime": "1804",
    "stopTime": "1877",
    "dateFormat": "yyyy",
    "fromContinuous": "nearest",
    "showInChartPanel": true,
    "disableDateTimeSelector": false,
    "timeProperty": "datestart, dateend"
  }`

I am unable to filter data based on the time line. Would really appreciate some help on configuring these items.

Thank You

**Comment by hardreddata** [comment]
Hi @sleagle 

For the WMTS you could try getting rid of the `parameters` group and see if it works like that, to begin. Also you need the full `URL`.

Factors like CORS can be an issue.

If this all works try changing the 900913 to 3857, or removing both of these lines . EPSG 3857 and 900913 are effectively the same thing, though the former looks a bit like _google_.

See for example https://gis.stackexchange.com/questions/189992/geowebcache-900913

You probably also want to move your layer up out of the parameters. See some examples https://github.com/TerriaJS/terriajs/blob/main/wwwroot/test/init/wms.json

It is looking good here https://map.terria.io/#share=s-aKXsWJLcgCW3cm35XT9e37GStDS though I cheated and added web data.

I am not sure about the time dependency.

Cheers.
